mrp_production_xls_report/README.rst

```diff
@@ -0,0 +1,45 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=========================
+Mrp Production Report Xls
+=========================
+
+This module extends the functionality of MEP to provide Mrp Production report in landscape mode.
```

Comment:
This module extends the functionality of MRP and provides XLS Production report in landscape mode.